src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -445,15 +460,16 @@ protected override int MakeSlot(BoundExpression node)
                     }
                     break;
                 case BoundKind.ConditionalReceiver:
-                    if (_lastConditionalAccessSlot != -1)
                     {
                         int slot = _lastConditionalAccessSlot;
                         _lastConditionalAccessSlot = -1;
                         return slot;
                     }
-                    break;
+                default:
+                    return base.MakeSlot(node);
```

Comment:
>return base.MakeSlot(node); [](start = 20, length = 27)

This used to be called for all cases that don't ```return```, now we always return ```-1``` for scenarios like that. Is this the right thing to do?

Reply:
Yes, I believe so. There is no overlap between the cases here and in the base.

---
In reply to: [246946078](https://github.com/dotnet/roslyn/pull/32304#discussion_r246946078) [](ancestors = 246946078)